### ISSUE TITLE
mBART50 finetuned many-to-many model failed to generate translation

### ISSUE THREAD

**Issue by Mao-KU** [huggingface/transformers#13903]
@patrickvonplaten
Hello, I downloaded mBART50 finetuned many-to-many model. When generating translations, it just repeats the same word over and over again.
This problem is also reported in https://github.com/huggingface/transformers/issues/7060#issuecomment-765648270.

I used mBART50 finetuned many-to-many model to do the ar_AR-en_XX translation on the IWSLT17 ar-en test set.
No bug occurred during the generation.
Also, I have done the multilingual fine-tuning for mBART25 successfully, so I suppose I did the preprocessing correctly.

The file that mBART50 fine-tuned N-N generated is put here:
https://lotus.kuee.kyoto-u.ac.jp/~zhuoyuanmao/ar_AR-en_XX

**Comment by Mao-KU** [comment]
sorry, I will go to fairseq to ask this question
